pkg/kubelet/kuberuntime/security_context_others.go

```diff
@@ -35,16 +35,16 @@ func verifyRunAsNonRoot(pod *v1.Pod, container *v1.Container, uid *int64, userna
 
 	if effectiveSc.RunAsUser != nil {
 		if *effectiveSc.RunAsUser == 0 {
-			return fmt.Errorf("container's runAsUser breaks non-root policy")
+			return fmt.Errorf("container's runAsUser breaks non-root policy (pod: %s, container: %s)", pod.Name, container.Name)
```

Comment:
ass pod namespace. `format.Pod` would be the best

Reply:
same below

Reply:
We have a standard utility for this: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/util/format/pod.go#L32

Reply:
done~

Reply:
nice , didnt know about this . Curious why is this utility in pkg/kubelet, it can be useful outside of kubelet as well

Reply:
> nice , didnt know about this . Curious why is this utility in pkg/kubelet, it can be useful outside of kubelet as well

I think the idea is to eventually switch to structured logging where this would be solved out of the box